esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2335,6 +2382,7 @@ bool CFileSprayEx::onDespray(IEspContext &context, IEspDespray &req, IEspDespray
         PROGLOG("Despray %s", srcname);
         double version = context.getClientVersion();
         const char* destip = req.getDestIP();
+        const char* destGroup = req.getDestGroup();
```

Comment:
is destGroup always a plane in container mode?

Reply:
Yes